Content.Server/Implants/ImplanterSystem.cs

```diff
@@ -83,6 +88,21 @@ private void OnImplanterAfterInteract(EntityUid uid, ImplanterComponent componen
         }
         //SS220-mindslave end
 
+        // SS220-fakeMSfix begin
+        if (component.Implant == FakeMindShieldImplant)
+        {
+            if (HasComp<MindShieldComponent>(target))
+            {
+                _popup.PopupEntity(Loc.GetString("mindslave-target-mindshielded"), args.User);
+                return;
+            }
+            if (HasComp<MindSlaveComponent>(target))
+            {
+                _popup.PopupEntity(Loc.GetString("mindslave-target-mindshielded"), args.User);
+                return;
+            }
```

Comment:
Судя по коду выше, лучше сделать
```suggestion
            if (_mindslave.IsEnslaved(target))
            {
                _popup.PopupEntity(Loc.GetString("mindshield-target-mindslaved"), target, args.User);
                return;
            }
```